backend/src/Entity/StoreOwnerProfileEntity.php

```diff
@@ -54,9 +56,21 @@ class StoreOwnerProfileEntity
     #[ORM\Column(type: 'string', length: 255, nullable: true)]
     private $stcPay;
 
-    #[ORM\Column(type: 'integer', nullable: true)]
+    #[ORM\Column(type: 'string', nullable: true)]
```

Comment:
We may need to set the length of the var _employeeCount_

Reply:
The work on this field is not finished, we do not know what the next requirements will be.